Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityGGVHand.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+
+namespace Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input
+{
+    /// <summary>
+    /// A Windows Mixed Reality Controller Instance.
+    /// </summary>
+    [MixedRealityController(
+        SupportedControllerType.GGVHand,
+        new[] { Handedness.Left, Handedness.Right })]
+    public class WindowsMixedRealityGGVHand : WindowsMixedRealityController
```

Comment:
This should also be encapsulated in the assessment of whether or not hands should directly inherit from controller logic.